src/services/aragon-sdk/aragon-sdk-service.api.ts

```diff
@@ -51,6 +51,13 @@ export interface IFetchMemberParams {
   blockNumber?: number;
 }
 
+export interface IFetchCreatorProposalsParams {
+  pluginAddress: string;
+  address: string;
+  pluginType: PluginTypes | undefined;
```

Comment:
Just to follow how it is currently defined on other param types 
```suggestion
  pluginType?: PluginTypes;
```